src/odemis/acq/feature.py

```diff
@@ -48,6 +49,96 @@
 MILLING_TASKS_PATH = os.path.join(os.path.dirname(milling_tasks_file), "milling_tasks.yaml")
 REFERENCE_IMAGE_FILENAME = "Reference-Alignment-FIB.ome.tiff"
 
+FIDUCIAL, POI, SURFACE_FIDUCIAL, PROJECTED_FIDUCIAL, PROJECTED_POI = (
+    "Fiducial",
+    "PointOfInterest",
+    "SurfaceFiducial",
+    "ProjectedPoints",
+    "ProjectedPOI",
+)
+
+
+class FIBFMCorrelationData:
```

Comment:
maybe you can separate FIB data and FM data and then use them in this class to separate concerns, upto you

Reply:
It is a dictionary to access different parameter settings for correlation between FIB and FM images. I am not sure if it would be useful to divide it further.